assignments/callbacks.js

```diff
@@ -24,28 +24,56 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(items.length);
 }
 
+getLength(items, function(length) {
+  console.log("LENGTH:", length);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(`LAST ITEM: ${items[3]}`);
```

Comment:
You should write this function as if you don't know what array it's taking in. That means that arr, not items, should be passed to the cb and you should be using the arrays length - 1 to get the last item.
```suggestion
  return cb(`LAST ITEM: ${arr[arr.length - 1]}`);
```